spark/src/main/scala/org/apache/comet/CometExecIterator.scala

```diff
@@ -92,7 +92,8 @@ class CometExecIterator(
       memoryLimitPerTask = getMemoryLimitPerTask(conf),
       taskAttemptId = TaskContext.get().taskAttemptId,
       debug = COMET_DEBUG_ENABLED.get(),
-      explain = COMET_EXPLAIN_NATIVE_ENABLED.get())
+      explain = COMET_EXPLAIN_NATIVE_ENABLED.get(),
+      sparkConfig = SparkEnv.get.conf.getAll.toMap)
```

Comment:
Doesn't `toMap` return a Scala Map (and not a Java HashMap)? Does this translate correctly to a JMap?
Also, `SparkConf.getAll` will return all Spark confs. Should we filter based on Comet conf and Spark confs of interest only?

Reply:
Comments are super valid and everything done in local branch yes, haven't yet pushed it

Reply:
I commented because I thought we were planning to get this in the 0.8.0 release :)